magefiles/magefile.go

```diff
@@ -499,6 +499,16 @@ func setRequiredEnvVars() error {
 			if !strings.Contains(jobName, "rehearse") {
 				os.Setenv(fmt.Sprintf("%s_CATALOG_URL", envVarPrefix), fmt.Sprintf("https://github.com/%s/%s", pr.RemoteName, pr.RepoName))
 				os.Setenv(fmt.Sprintf("%s_CATALOG_REVISION", envVarPrefix), pr.CommitSHA)
+				if isPRPairingRequired("release-service") {
+					os.Setenv(fmt.Sprintf("%s_IMAGE_REPO", envVarPrefix),
+						"quay.io/redhat-user-workloads/rhtap-release-2-tenant/release-service/release-service")
+					pairedSha := getPairedCommitSha("release-service")
+					if pairedSha != "" {
+						os.Setenv(fmt.Sprintf("%s_IMAGE_TAG", envVarPrefix), fmt.Sprintf("on-pr-%s", pairedSha))
+					}
+					os.Setenv(fmt.Sprintf("%s_PR_OWNER", envVarPrefix), pr.RemoteName)
+					os.Setenv(fmt.Sprintf("%s_PR_SHA", envVarPrefix), pr.CommitSHA)
```

Comment:
I guess this env var refers to the [env var from here](https://github.com/redhat-appstudio/infra-deployments/blob/0c3717273b6aeb4b14b114a185bca4468433ec07/hack/preview-template.env#L61), so should it maybe get the value from `pairedSha` instead?
```suggestion
					os.Setenv(fmt.Sprintf("%s_PR_SHA", envVarPrefix), pairedSha)
```

Reply:
Yes, nice catch thanks